src/index.js

```diff
@@ -28,6 +28,18 @@ function iPipeTo(
 		return str.length > maxWidth ? str.substr(0, maxWidth) + "..." : str;
 	}
 
+	function getDefaultChoices(promptType) {
+		if (promptType.type === 'list') {
+			return options.default;
+		}
+
+		if (promptType.type === 'checkbox') {
+			return options.default
+				.split(' ')
```

Comment:
I agree, I'll make some changes and add this extra option :)

Reply:
I think it would make more sense if the default value of the separator is similar to [this line](https://github.com/ruyadorno/ipt/blob/master/src/cli.js#L68) using `os.EOL` (e.g. `\n` in Unix).

This way you can easily pass file contents, separated by new lines.

Reply:
Also, do you think it is necessary with an alias? There's not letters that matches with the naming `default-separator`. I tried using `P`, but it seemed too unclear.